internal/file/tar.go

```diff
@@ -93,3 +95,13 @@ func UnTarGz(dst string, r io.Reader) error {
 		}
 	}
 }
+
+func copyWithLimits(writer io.Writer, reader io.Reader, byteReadLimit int64, target string) error {
+	// limit the tar reader to 5GB per file to prevent decompression bomb attacks
```

Comment:
I think the code would be more clear without this comment, actually. It looks like the "5GB" value has been parametrized, not set explicitly in this scope. And the comment above for `decompressionByteReadLimit` does a nice job of explaining this value's motivation 🙌

Reply:
looks like I migrated the code without updating the comment --I'll update